frontend/package-lock.json

```diff
@@ -3384,8 +3384,7 @@
     "binary-extensions": {
```

Comment:
Do we need to make changes to `package-lock`?
We're not adding/removing a dependency in this PR are we?

I believe this is a mismatch between user's preinstalled packages and the repo's. (ping @bodhisha that's what I meant when I said package-lock is a mess xD)